src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -219,7 +429,15 @@ internal ResponseContext(string privilege)
 
             internal void FireEvent(CallCause _cause, RequestResult _result)
             {
-                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs() { cause = _cause, result = _result, privilege = _privilege });
+                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs { cause = _cause, result = _result, privilege = _privilege });
+            }
+
+            /// <summary>
+            /// ResponseContext class destructor
+            /// </summary>
+            ~ResponseContext()
+            {
+                s_responseMap.Remove(_privilege);
```

Comment:
It is wrong
if contain this object in s_responseMap, it never called destructor. 
So, you does not need to implement this destructor

Reply:
Done, I have removed it.